app/src/test/java/org/simple/clinic/bp/history/adapter/BloodPressureHistoryListItemTest.kt

```diff
@@ -0,0 +1,81 @@
+package org.simple.clinic.bp.history.adapter
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.patient.PatientMocker
+import org.simple.clinic.util.TestUtcClock
+import org.threeten.bp.Duration
+import org.threeten.bp.Instant
+import org.threeten.bp.temporal.ChronoUnit.MINUTES
+import java.util.UUID
+
+class BloodPressureHistoryListItemTest {
+
+  @Test
+  fun `if blood pressure is editable, then user can edit blood pressure`() {
```

Comment:
Reword this test?

Ex: `if the blood pressure was created within the BP editable duration, then the bp list item must be editable`